sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeOperationExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// </summary>
+    public static class RecognizeOperationExtensions
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="operation"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public static async ValueTask<Response<IReadOnlyList<RecognizedReceipt>>> WaitForCompletionAsync(this Task<RecognizeReceiptsOperation> operation, CancellationToken cancellationToken = default)
```

Comment:
Extension method accepts `Task` but returns `ValueTask`. Why `FormRecognizer` methods return `Task`?

Reply:
I'll update these to return `Task` to maintain consistency with `FormRecognizerClient`.